RevenueCatUI/CustomerCenter/ViewModels/ManageSubscriptions/ManageSubscriptionsViewModel.swift

```diff
@@ -258,4 +258,55 @@ private extension CustomerCenterConfigData.Screen {
 
 }
 
+private extension Array<CustomerCenterConfigData.HelpPath> {
+    func relevantPathsForPurchase(
+        _ purchaseInformation: PurchaseInformation?,
+        now: Date) -> [CustomerCenterConfigData.HelpPath] {
+
+        guard let purchaseInformation else {
+            return self
+        }
+
+        return filter { !purchaseInformation.isLifetime || $0.type != .cancel }
+                .filter { $0.refundWindow.map { $0.withinWindow(purchaseInformation, now: now) } ?? true }
+    }
+}
+
+private extension CustomerCenterConfigData.HelpPath.RefundWindowDuration {
+    func withinWindow(_ purchaseInformation: PurchaseInformation, now: Date) -> Bool {
+        switch self {
+        case .forever:
+            return true
+
+        case let .duration(duration):
+            return duration.withinDuration(referenceDate: purchaseInformation.latestPurchaseDate, now: now)
+
+        @unknown default:
+            return true
+        }
+    }
+}
+
+private extension ISODuration {
+    func withinDuration(referenceDate: Date?, now: Date) -> Bool {
+        guard let referenceDate else {
+            return true
+        }
+
+        var dateComponents = DateComponents()
+        dateComponents.year = self.years
+        dateComponents.month = self.months
+        dateComponents.weekOfYear = self.weeks
+        dateComponents.day = self.days
+        dateComponents.hour = self.hours
+        dateComponents.minute = self.minutes
+        dateComponents.second = self.seconds
+
+        let calendar = Calendar.current
+        let endDate = calendar.date(byAdding: dateComponents, to: referenceDate) ?? referenceDate
+
+        return referenceDate < endDate && now <= endDate
```

Comment:
I wonder if we should compare agains the `requestDate` of the `CustomerInfo` instead. This is easily hackable by changing the device time. The `requestDate` comes from the backend so it's not as easy to modify. Probably not worth the effort, but what do you think?